package.json

```diff
@@ -12,6 +12,15 @@
   },
   "dependencies": {
     "com.unity.nuget.newtonsoft-json": "3.1.0",
-    "com.unity.textmeshpro": "3.0.6"
-  }
+    "com.unity.textmeshpro": "3.0.6",
+    "com.unity.inputsystem": "1.7.0",
+    "com.unity.cinemachine": "2.9.7"
+  },
+  "samples": [
+    {
+      "displayName": "Third Person Demo (URP)",
+      "description": "This third person demo is a sample Unity game demoing Regression Game’s Unity SDK. It uses Unity's Universal Render Pipeline (URP). To make sure shaders appear correctly, please make sure your project is setup with URP before downloading sample.",
```

Comment:
I think you missed the word `the` or `this` at the end of this block `... with URP before downloading [the/this] sample.`